zerver/lib/onboarding.py

```diff
@@ -92,8 +92,16 @@ def send_initial_pms(user: UserProfile) -> None:
         getting_started_url="/help/getting-started-with-zulip",
     )
 
+    # Note: Welcome bot doesn't trigger email/push notifications,
+    # as this is intended to be seen contextually in the application.
+    # Set 'disable_external_notifications = True' for the implementation of
+    # this rule.
```

Comment:
You might want to remove this part of the comment -- the code itself is pretty clear :)
(Same for the comment in `send_welcome_bot_response`)